src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector/nuxeo-aspera-release-notes/nuxeo-aspera-connector-2021-3-1.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Nuxeo Aspera 2021.3.1
+description: Release notes for Nuxeo Aspera Connector 2021.3.1
+tree_item_index: 100
+review:
+  comment: ''
+  date: '2022-07-01'
+  status: ok
+toc: true
+---
+
+
+## Released Changes
+
+
```

Comment:
- The author and Last Contributor name set as "System" in the Aspera folder upload has been changed to the actual user<br/>[[NXP-30867](https://jira.nuxeo.com/browse/NXP-30867)]
- Aspera download from collection is now possible<br/>[[NXP-30862](https://jira.nuxeo.com/browse/NXP-30862)]
- Manual retry mechanism implemented for failed Aspera upload complete transfers<br/>[[NXP-30965](https://jira.nuxeo.com/browse/NXP-30965)